### PR TITLE
refactor: consolidate ClientBrowserAdapter into BrowserAdapter

### PR DESCRIPTION
**Pull request by dbjorge**
#### Description of changes

Per past engineering discussion on BrowserAdapter design and as a precursor to trying a switch from chrome.* to browser.* APIs to ease firefox support, consolidating the ClientBrowserAdapter/ClientChromeAdapter classes into the BrowserAdapter/ChromeAdapter classes.

Included moving the browser-adapters into common/ to account for new usage points and updating related variable names.

Out of scope:
* Similarly consolidating the DevToolsChromeAdapter family of classes
* Consolidation of other browser adapter interfaces

#### Pull request checklist

- [n/a] Addresses an existing issue: Fixes #0000
- [n/a] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/Devtools/dev-tool-initializer.ts [hunk 1]
```diff
@@ -7,15 +7,15 @@ import { DevToolState } from '../common/types/store-data/idev-tool-state';
 import { InspectHandler } from './inspect-handler';
 
 export class DevToolInitializer {
-    protected _chromeAdapter: DevToolsChromeAdapter;
+    protected _devToolsChromeAdapter: DevToolsChromeAdapter;
```
Comment (smoralesd) [inline src/Devtools/dev-tool-initializer.ts]: nitpick: should we drop the `_` on the name?

File: src/Devtools/dev-tool-initializer.ts [hunk 2]
```diff
@@ -7,15 +7,15 @@ import { DevToolState } from '../common/types/store-data/idev-tool-state';
...
 
-    constructor(chromeAdapter: DevToolsChromeAdapter) {
-        this._chromeAdapter = chromeAdapter;
+    constructor(devToolsChromeAdapter: DevToolsChromeAdapter) {
```
Comment (smoralesd) [inline src/Devtools/dev-tool-initializer.ts]: nitpick: should we use parameter properties here? i.e.
```typescript
constructor(protected readonly devToolsChromeAdapter: DevToolsChromeAdapter) {
`````

Comment (dbjorge) [reply src/Devtools/dev-tool-initializer.ts]: Agree with this, but will address in separate PR

File: src/Devtools/dev-tool-initializer.ts (hunk 1 above)
Comment (dbjorge) [reply src/Devtools/dev-tool-initializer.ts]: Agree with this, but will address in separate PR

**Pull request merged**